loleaflet/src/control/Control.JSDialogBuilder.js

```diff
@@ -2482,7 +2482,6 @@ L.Control.JSDialogBuilder = L.Control.extend({
 
 			if (twoPanelsAsChildren) {
 				handler = this._controlHandlers['paneltabs'];
-				processChildren = handler(childObject, childData.children, this);
```

Comment:
Doesn't handler() have side-effects? I expect it does.

Notice that by removing the call the variable 'handler' is not user after the assignment.

Reply:
I just want to remove variable. Actually, I did not look at side-effects.

Reply:
Perhaps best to restore the handler call?